### PR TITLE
Added 'man and Git for Windows' callout recommending use of Google search since 'man' command isn't supported.

### PR DESCRIPTION
**Pull request by mikej888**

### CODE REVIEW

File: 06-find.md [hunk 1]
```diff
@@ -160,6 +160,14 @@ To find out what they are, we can type `man grep`.
 it prints a description of a command and its options,
 and (if you're lucky) provides a few examples of how to use it.
 
+> ## `man` and Git for Windows {.callout}
+>
+> The bash shell provided by Git for Windows does not
+> support the `man` command. Googling for
+> `unix man page COMMAND` (e.g. `unix man page grep`)
+> provides links to numerous copies of the Unix manual
+> pages online.
```
Comment (shwina) [inline 06-find.md]: What about `--help` for quicker/offline reference? Works for most commands on Windows, but I haven't tested personally..

Comment (mikej888) [inline 06-find.md]: I've updated this page and reference.md with information on the --help flag and how it is supported by many bash commands, and programs people have written.

**Comment by shwina** [comment]
Can we point to a specific resource(s) instead of just saying "Google this"? For example, http://www.gnu.org/software/grep/manual/

**Comment by gdevenyi** [comment]
Looks great thanks!

**Pull request merged**